server/service/client_software.go

```diff
@@ -42,7 +43,11 @@ func (c *Client) ApplyNoTeamSoftwareInstallers(softwareInstallers []fleet.Softwa
 func (c *Client) applySoftwareInstallers(softwareInstallers []fleet.SoftwareInstallerPayload, query url.Values, dryRun bool) ([]fleet.SoftwarePackageResponse, error) {
 	path := "/api/latest/fleet/software/batch"
 	var resp batchSetSoftwareInstallersResponse
-	if err := c.authenticatedRequestWithQuery(map[string]interface{}{"software": softwareInstallers}, "POST", path, &resp, query.Encode()); err != nil {
+	log.Default().SetPrefix("[JVE_LOG] ")
+	for _, si := range softwareInstallers {
+		log.Default().Printf("would send sha256 %s to Fleet", si.SHA256)
+	}
+	if err := c.authenticatedRequestWithQuery(map[string]any{"software": softwareInstallers}, "POST", path, &resp, query.Encode()); err != nil {
```

Comment:
Why did this chage?

Reply:
Probably just formatting? Not sure.

Reply:
Swapped from empty interface to any fwiw

Reply:
Ohh yeah, those are equivalent as of some relatively recent Go version. `any` is the preferred way now according to the Go team.